.travis.yml

```diff
@@ -26,12 +26,12 @@ matrix:
       sudo: required
       language: generic
 install:
-  - pip3 install .[dev]
+  - pip3 install .[testing]
 script:
   - pytest
-  - if [[ "$RUN_MYPY" == "yes" ]]; then pip install -r requirements.txt && ./tools/run-mypy; fi
-  - if [[ "$RUN_ISORT_CHECK" == "yes" ]]; then pip install -r requirements.txt && ./tools/run-isort-check; fi
-  - if [[ "$RUN_PYCODESTYLE" == "yes" ]]; then pip install -r requirements.txt && pycodestyle; fi
+  - if [[ "$RUN_MYPY" == "yes" ]]; then pip3 install .[linting] && ./tools/run-mypy; fi
```

Comment:
The deps on Travis are installed by copying Travis cache, which is fast. I doubt the speedup is substantial? How much is the speedup anyway?

Reply:
Are you referring to the speedup from this particular change? I agree that they should be cached, but this is also for clarity of which packages are installed - I'll update that.

Reply:
Yes, for this particular change. the `pipenv` -> `pip3` is definitely a speedup. It's not a big issue though.